modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -3953,6 +3953,7 @@ create-new-test=Create New Test
 create-new-variant=Create New Variant
 create-page-template=Create Page Template
 create-page-x=Create Page '{0}'
+create-personalized-variations-of-the-collections-for-different-segments=Create personalized variations of the collections for different segments. The order on the list affects the priority of the variations. The most priority on top.
```

Comment:
@beltranrengifo `The highest priority on top` sounds better. Check with Marcia.

Reply:
@beltranrengifo I like Álex suggestion as well

Reply:
Already fixed